app/src/main/java/com/gujja/ajay/fourthver/Adapters/Child_Age_Adapter.java

```diff
@@ -29,8 +29,8 @@
 
     //List of Images
     private int[] ist_images = {
-            R.drawable.hungry_fox_logo, //
-            R.drawable.lion_logo,
+            R.drawable.cat_dog_logo, //
```

Comment:
Cat_Dog logo Background needs to be removed.

Reply:
okay !!!